includes/class-wp-user-manager.php

```diff
@@ -0,0 +1,488 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected vipgoci-phpcs-scan-iugbss.php, but found vipgoci-phpcs-scan-iUgbSs.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).